src/main/java/com/aws/greengrass/status/FleetStatusService.java

```diff
@@ -485,7 +473,8 @@ private void uploadFleetStatusServiceData(Set<GreengrassService> greengrassServi
                 .build();
 
         publisher.publish(fleetStatusDetails, components);
-        logger.atInfo().event("fss-status-update-published").log("Status update published to FSS");
+        logger.atInfo().event("fss-status-update-published").log("Status update published to FSS with "
+                + "FleetStatusDetails {}", fleetStatusDetails);
```

Comment:
you could make this a key value instead

Reply:
Fair point. Will leave the original log event at Info and add  FleetStatusDetails as key value at Debug